### ISSUE TITLE
doc/web2/examples/auth/credsetup.py missing import

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1772]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)| @dreid reported|
|-|-|
|Trac ID|trac#1772|
|Type|defect|
|Created|2006-05-31 22:35:21Z|

doesn't import portal from cred.

Attachments:

* [credsetup.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/4c9/4c90181223e889a37c94b7c61243d3a5f6e0a8a4/6664856e7e492601836957d3ee7744851f6e8d2d.patch) (363 bytes) - added by dreid on 2006-05-31 22:35:55Z - fix missing import
* [http-auth2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/4c9/4c90181223e889a37c94b7c61243d3a5f6e0a8a4/12b8d4f61a9f0d2037481a65a9e01d6e03dfcc61.diff) (1165 bytes) - added by dreid on 2006-05-31 23:04:53Z - also use make httpauth.tac use multiple schemes

<details><summary>Searchable metadata</summary>

```
trac-id__1772 1772
type__defect defect
reporter__dreid dreid
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web2 web2
keywords__documentation documentation
time__1149114921000000 1149114921000000
changetime__1149116959000000 1149116959000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)|@dreid set status to `closed`|
|-|-|

(In [16977]) apply patch to fix twisted.web2.auth examples.

Author: dreid
Reviewer: dialtone
fixes [#1772](https://github.com/twisted/twisted/issues/1772)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
